force_bdss/mco/optimizer_engines/weighted_optimizer_engine.py

```diff
@@ -55,15 +56,23 @@ def sen_scaling_method(dimension, weighted_optimize):
     return scaling_factors
 
 
-class WeightedOptimizerEngine(ScipyOptimizerEngine):
-    """ Performs local optimization of multiobjective function using scipy.
-    The multiobjective function is converted to a scalar by dot product
-    with a weights vector (`weighted_score`).
+class WeightedOptimizerEngine(BaseOptimizerEngine, ScipyOptimizer):
+    """ Performs multi-objective optimization by weighting.
     """
 
```

Comment:
I had some extensive docstring here, but the doc build complained so I took it out ;(

Reply:
This might be due to the use of underscores to signify a title word that isn't recognised (Sphinx expects specific title words like `Parameters` and `Returns` in docstrings). It looks like the doc build is still complaining for `i_optimizer.py` because of this

Try using ~~~~~ to underscore the test instead, rather than -------